Source/WebCore/dom/NodeRareData.h

```diff
@@ -305,10 +308,15 @@ class NodeRareData {
     void operator delete(NodeRareData*, std::destroying_delete_t);
 
 private:
+    struct DataBitFields {
+        uint16_t isElementRareData : 1;
```

Comment:
Should be `bool`, right? We don't want to use 16 bits for this bitfield.

Reply:
Done and the bit fields structure was removed.